templater/jobs/postsubmits.go

```diff
@@ -0,0 +1,483 @@
+			ImageBuild:         true,
+			Resources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "4",
+					Memory: "16Gi",
+				},
+			},
+			BuildkitResources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "1",
+					Memory: "4Gi",
+				},
+			},
+			EnvVars: []*EnvVar{
+				{
+					Name:  "TEST_ROLE_ARN",
+					Value: "arn:aws:iam::125833916567:role/TestBuildRole",
+				},
+				{
+					Name:  "ARTIFACT_BUCKET",
+					Value: "eks-d-postsubmit-artifacts",
+				},
+				{
+					Name:  "RELEASE_BRANCH",
+					Value: "1-X",
+				},
+				{
+					Name:  "RELEASE_VARIANT",
+					Value: "minimal",
+				},
+				{
+					Name:  "CONTROL_PLANE_INSTANCE_PROFILE",
+					Value: "arn:aws:iam::125833916567:instance-profile/KopsControlPlaneBuildRole",
+				},
+				{
+					Name:  "NODE_INSTANCE_PROFILE",
+					Value: "arn:aws:iam::125833916567:instance-profile/KopsNodesBuildRole",
+				},
+				{
+					Name:  "KOPS_STATE_STORE",
+					Value: "s3://testbuildstack-125833916-kopsbuildstatestorebucke-d4esen60nfrk",
+				},
+				{
+					Name:  "IMAGE_REPO",
+					Value: "public.ecr.aws/h1r8a7l5",
+				},
+				{
+					Name:  "DOCKER_CONFIG",
+					Value: "/home/prow/go/src/github.com/aws/eks-distro/.docker",
+				},
+			},
+		},
+		"dev-release-1-X-postsubmits.yaml": JobConfig{
+			HasReleaseBranches: true,
+			JobName:            "dev-release-1-X-postsubmit",
+			RunIfChanged:       "release/1-X/development/RELEASE",
+			MaxConcurrency:     1,
+			Timeout:            "4h",
+			Commands:           []string{"build/lib/buildkit_check.sh", "./release/prow.sh", "touch /status/done"},
+			ImageBuild:         true,
+			Resources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "4",
+					Memory: "16Gi",
+				},
+			},
+			BuildkitResources: &Resources{
+				Requests: &ResourceConfig{
+					CPU:    "1",
+					Memory: "4Gi",
+				},
+			},
+			EnvVars: []*EnvVar{
+				{
+					Name:  "AWS_REGION",
+					Value: "us-east-1",
+				},
+				{
+					Name:  "ARTIFACT_BUCKET",
+					Value: "eks-d-postsubmit-artifacts",
+				},
+				{
+					Name:  "RELEASE_BRANCH",
+					Value: "1-X",
+				},
+				{
+					Name:  "IMAGE_REPO",
+					Value: "public.ecr.aws/h1r8a7l5",
+				},
+			},
+		},
+		"eks-distro-docs-postsubmits.yaml": JobConfig{
+			JobName:            "eks-distro-docs-postsubmit",
+			RunIfChanged:       "docs/.*",
+			ServiceAccountName: "docs-build-account",
+			Commands:           []string{"make release -C docs", "mv ./docs/site/* /logs/artifacts"},
+		},
+		"prod-release-1-X-postsubmits.yaml": JobConfig{
+			HasReleaseBranches: true,
+			JobName:            "prod-release-1-X-postsubmit",
+			RunIfChanged:       "release/1-X/production/RELEASE",
+			MaxConcurrency:     1,
+			Timeout:            "4h",
+			ServiceAccountName: "release-build-account",
+			Commands:           []string{"build/lib/buildkit_check.sh", "./release/prow-release.sh", "touch /status/done"},
```

Comment:
Even like here, I wouldnt mind turning this into a make target in the repo to run the prow-release.sh instead of calling it directly here